src/metabase/util/honeysql_extensions.clj

```diff
@@ -293,6 +293,7 @@
 (defn ->date                     "CAST `x` to a `date`."                     [x] (maybe-cast :date x))
 (defn ->datetime                 "CAST `x` to a `datetime`."                 [x] (maybe-cast :datetime x))
 (defn ->timestamp                "CAST `x` to a `timestamp`."                [x] (maybe-cast :timestamp x))
+(defn ->timestamptz              "CAST `x` to a `timestamptz`."              [x] (maybe-cast :timestamptz x))
```

Comment:
I think only Postgres has a type called `timestamptz` so I don't think this really makes sense inside a general namespace like this for use in a bunch of different places. It seems like it belongs in a Postgres-specific namespace

Reply:
Updated in https://github.com/metabase/metabase/pull/25722/commits/4a5c1eecb488a8beffcf31b16247356998d6ae6c